### DISCUSSION TITLE
Are these NPM errors due to Zigbee2MQTT?

### DISCUSSION THREAD

**Discussion by manelrodero** [Koenkk/zigbee2mqtt#23421] [General]
Hello,

I just migrated my Zigbee2MQTT installation on Raspberry Pi 4 + Docker to Proxmox + LXC.

I have used Helper Scripts to install Zigbee2MQTT and they download the code from GitHub.

These errors appear about NPM modules that are deprecated and also that a new version of NPM is available.

![image](https://github.com/user-attachments/assets/95f4e242-a2a5-474d-a3c8-272e9458cb51)

Is this because of the Zigbee2MQTT project? If so, do they need to be updated? How would it be done?

Thank you.

**Comment by francisp2** [comment]
See here: 
https://github.com/Koenkk/zigbee2mqtt/issues/23238#issuecomment-2204103062

**Comment by manelrodero** [comment]
Thank you. I don't understand why I hadn't found that issue.

Since they are indirect dependencies, wouldn't we have to look for the module that we use to notify the developer of it?

If not, we could have something dangerous running (either due to a memory leak or worse).
